src/clr-addons/quick-list/quick-list.ts

```diff
@@ -4,34 +4,56 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import {
+  AfterViewChecked,
+  Component,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+  QueryList,
+  ViewChildren,
+} from '@angular/core';
 import { CLR_BLANK_OPTION, ClrQuickListValue } from './add-option.service';
+import { ClrAddOption } from './add-option';
 
 @Component({
   selector: 'clr-quick-list',
   host: { '[class.quick-list]': 'true' },
   templateUrl: './quick-list.html',
 })
-export class ClrQuickList<T> implements OnInit {
+export class ClrQuickList<T> implements OnInit, AfterViewChecked {
   @Input('clrBlankOption') blankOption: ClrQuickListValue<T> = CLR_BLANK_OPTION;
   @Input('clrAllValues') allValues: Array<ClrQuickListValue<T>> = [this.blankOption];
   @Input('clrMandatory') mandatory: boolean = false;
   @Input('clrValues') values: Array<ClrQuickListValue<T>> = [];
   @Input('clrAddLabel') addLabel: string = 'ADD';
-  @Input('clrControlClasses') controlClasses = 'clr-col-md-10';
```

Comment:
i would not recommend removing this, as it's a breaking change and the label also provides md-2 as default, so the layout should be fine without developer-interaction.

Reply:
if i provide this as a default, the field will not be usable in vertical forms as this classes add a padding to the field